packages/legend-graph/src/graphManager/AbstractPureGraphManager.ts

```diff
@@ -217,8 +218,10 @@ export abstract class AbstractPureGraphManager {
   ): Promise<string>;
 
   // ------------------------------------------- Test  -------------------------------------------
+
   abstract runTests(
     graph: PureModel,
+    graphManagerState: GraphManagerState,
```

Comment:
No, this doesn't look right. We must not pass any `state/store` here, wrong layer

Reply:
See my note in `Testable.ts` so we can pass in more appropriate parameters here